### PR TITLE
Feature 963 quick search

### LINKED ISSUE #963: Fix documentation quick search links
The quick search page (docs/Users_Guide/quicksearch.rst and use case documentation files (docs/use_cases) have hard-coded URLs to perform the search. They point to the develop version of the docs. These have to be updated when a new release is created. This also prevents testing that links work properly in a branch because the changed content doesn't exist in develop yet.

**Some of the URLs in the use case doc files still use dtcenter.github.io URLs which are broken!**

## Describe the Enhancement ##
Some thought/investigation is needed to come up with a good solution. My initial thoughts:

* URLs can be relative instead of a full hard-coded path
* URLs that start with / are relative to the "root" of the webpage. We should investigate what the "root" of a ReadTheDocs page is. Typically the root is the top level, i.e. metplus.readthedocs.io but we need to know the rest of the URL for the current branch/tag/version, i.e. metplus.readthedocs.io/en/develop. If ReadTheDocs somehow sets the root to be this location, then we can easily create paths relative to that for the searches. I think this would be the ideal solution if that is the case.
* URLs that start with ../ are relative to the current page. This would work great for the quicksearch.rst page, however the use case doc pages will have a different relative path. We typically copy links from the quicksearch page and paste them into the use case doc files (and modify the link text to match the actual keyword) so this would involve an extra step to add extra "../" to get the correct relative path. One option is to remove the links from the use case doc pages since the actual keyword text is the only required text so that the search finds the page. The URL on those pages are just for convenience to search for other related cases. We could add a link to the quick search page so that users can easily get to that page to search for that keyword.

### Time Estimate ###
*Estimate the amount of work required here.*
*Issues should represent approximately 1 to 3 days of work.*

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [ ] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Linked issues**
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by lisagoodrich**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Working with @jprestop.  We replaced 
`https://metplus.readthedocs.io/en/develop`
with
`..`
This linked to the current working branch directory instead of the 'develop' branch

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
I was not able to get the "feature_963_quick_search branch to pull up in readthedocs.  But the develop branch is still showing up.  Please confirm this is correct.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[Fill in date]**.</br>
6/30/21

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization**. HELP!!!  I couldn't find Organization.  I don't know where this is located.

 level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select **Linked issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

**Review by jprestop — APPROVED** [review]
I reviewed the changes for typos.  In the documentation for the feature branch page:
https://metplus.readthedocs.io/en/feature_963_quick_search/Users_Guide/quicksearch.html
I can click on the quick search links and see that they go to the feature branch pages.  I approve this request.

We will leave this issue open, as there is more work to do, but we wanted to get this change in quickly.

**Comment by lisagoodrich** [comment]
I'm leaving this branch open since there is more work to be done on this issue.

File: docs/Users_Guide/quicksearch.rst [hunk 1]
```diff
@@ -12,29 +12,31 @@ METplus Quick Search for Use Cases
 Use Cases by MET Tool:
 ======================
 
+.. |branch| replace:: feature_963_quick_search
```
Comment (georgemccabe) [inline docs/Users_Guide/quicksearch.rst:15]: This line can be removed since it is no longer needed.

Comment (jprestop) [reply docs/Users_Guide/quicksearch.rst:15]: @georgemccabe Great catch!  Sorry I missed that.  Thanks!  I just removed that line in the feature branch.

**Pull request merged**